api/db/post.js

```diff
@@ -825,6 +847,18 @@ function _runScript(seriesBag, next) {
   var who = seriesBag.who + '|' + _runScript.name;
   logger.verbose(who, 'Inside');
 
+  /* jshint camelcase:false */
+  seriesBag.scriptEnvs = seriesBag.scriptEnvs || {};
```

Comment:
spawn does not copy over these envs from the enviroment. I had missed this in the initial design.

Reply:
I'll add this to all the spawns in a subsequent PR.